services/src/generated/main/grpc/io/grpc/instrumentation/v1alpha/MonitoringGrpc.java

```diff
@@ -0,0 +1,500 @@
+package io.grpc.instrumentation.v1alpha;
```

Comment:
We really don't want to be committing generated source, please take it out.  You can do it most easily by just adding this: https://github.com/mavrukin/grpc-java/blob/master/.gitignore#L31 to your .gitignore file and that will take care of it